### PR TITLE
Merging r1.2 back into master.

### PR DESCRIPTION
**Pull request by av8ramit**

### CODE REVIEW

**Review by martinwicke — APPROVED** [review]

**Review by jhseu — CHANGES_REQUESTED** [review]
This change deletes the non_max_suppression_v2 op. Let's keep it in master and just resolve it as part of b/62486586?

**Comment by jhseu** [comment]
In other words, omit 323d40fce604e9e31975714197ce4a7064b55dc2

**Comment by martinwicke** [comment]
yes please.​

**Review by allenlavoie — APPROVED** [review]
Either way works for the RewriterConfig PR; sync it or don't sync it, whatever's easiest. There's a CL coming the other direction which does about the same thing but includes a bit more explanation.

File: tensorflow/core/ops/image_ops.cc [hunk 1]
```diff
@@ -1030,50 +1030,11 @@ method: A string specifying the interpolation method. Only 'bilinear' is
 // --------------------------------------------------------------------------
 
 REGISTER_OP("NonMaxSuppression")
-  .Input("boxes: float")
```
Comment (jhseu) [inline tensorflow/core/ops/image_ops.cc]: This probably shouldn't be deleted. Not sure where it's coming from.

**Review by jhseu — APPROVED** [review]

**Pull request merged**